compiler/ballerina-parser/src/test/java/io/ballerinalang/compiler/parser/test/syntax/expressions/TableConstructorTest.java

```diff
@@ -117,9 +117,4 @@ public void testTableWithExtraToken() {
         test("table foo key() [{k1:v1}]", "table-constructor/table_constructor_assert_24.json");
         test("table foo [{k1:v1}]", "table-constructor/table_constructor_assert_25.json");
     }
-
-    @Test
-    public void testTableWithTwoKeySpecifiersWithExtraTokenInBetween() {
-        test("table key(a) foo key(b) []", "table-constructor/table_constructor_assert_26.json");
-    }
```

Comment:
This was removed from table constructor tests as this is recovered as a query expression. Similar test case has been added under query expression test cases :)